Formula/gotop.rb

```diff
@@ -1,14 +1,14 @@
 class Gotop < Formula
   desc "Terminal based graphical activity monitor inspired by gtop and vtop"
-  homepage "https://github.com/cjbassi/gotop"
-  url "https://github.com/cjbassi/gotop/archive/3.0.0.tar.gz"
-  sha256 "d5147080bb6057f0bf0900b38438e89aa066959c845bdd4c84a9c9fe478b176f"
+  homepage "https://github.com/xxxserxxx/gotop"
+  url "https://github.com/xxxserxxx/gotop/archive/3.3.0.tar.gz"
+  sha256 ""
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "e92acc3b1327317f9a5a6a09fe841d4cdf116ebeb3b225fa73d28b10a21d0170" => :catalina
-    sha256 "fbf79c06ef6ef84bd3f3f28fba3e49308e295f816facd0f6b7f696d6789e69a9" => :mojave
-    sha256 "58606da95a9fc75bfca7646e32e1f14e90fca4600b2bffddf5a7c14fbe3f3268" => :high_sierra
+    sha256 "" => :catalina
+    sha256 "" => :mojave
+    sha256 "" => :high_sierra
   end
```

Comment:
Please don't change these. They're autogenerated by our CI when we merge the PR.

Reply:
Good to know. Thanks! Does @xxxserxxx just need to version bump the URL, then?

Will the Homebrew CI then compute the sha256s for each?

Reply:
This whole block is handled by CI.

Reply:
@dawidd6 Okay, thanks! So, if you don't mind, could you check my understanding?

These two snippets need to be updated every time `gotop` has a version bump.
https://github.com/Homebrew/homebrew-core/blob/3ba0438784e7ccfa7e2bb3e5cde31516eb99b6ab/Formula/gotop.rb#L4

https://github.com/Homebrew/homebrew-core/blob/3ba0438784e7ccfa7e2bb3e5cde31516eb99b6ab/Formula/gotop.rb#L5

This snippet is governed by whatever dependencies @xxxserxxx introduces, __and are present in `homebrew-core`.__
https://github.com/Homebrew/homebrew-core/blob/3ba0438784e7ccfa7e2bb3e5cde31516eb99b6ab/Formula/gotop.rb#L14

This snippet is based on whatever installation steps `gotop` needs to become a system command.
https://github.com/Homebrew/homebrew-core/blob/3ba0438784e7ccfa7e2bb3e5cde31516eb99b6ab/Formula/gotop.rb#L17

The ensuing block is handled by Homebrew, not @xxxserxxx. (Don't seem able to highlight the actual block, sorry.)
https://github.com/Homebrew/homebrew-core/blob/3ba0438784e7ccfa7e2bb3e5cde31516eb99b6ab/Formula/gotop.rb#L7

Additionally, docs allude to `brew bump-formula-pr` as being the method of updating this particular Formula.

Reply:
Correct. `bump-formula-pr` is the standard way to update the formula easily.